src/encoded/static/components/cart/facet.js

```diff
@@ -614,78 +579,68 @@ FacetExpander.defaultProps = {
+    expanded,
+    expanderClickHandler,
+    selectedFacetTerms,
+    facetTermClickHandler,
+    clearFacetSelections,
+    options,
+}) => {
     /**
      * Handle a click in a facet term by calling a parent handler.
      * @param {string} term Clicked facet term
      */
-    handleFacetTermClick(term) {
-        this.props.facetTermClickHandler(this.props.displayedFacetField.field, term);
-    }
+    const handleFacetTermClick = (term) => {
+        facetTermClickHandler(displayedFacetField.field, term);
+    };
 
     /**
      * Handle a click in the facet expander by calling a parent handler.
      * @param {object} e React synthetic event
      */
-    handleExpanderEvent(e) {
-        this.props.expanderClickHandler(this.props.displayedFacetField.field, e.altKey);
-    }
+    const handleExpanderEvent = (e) => {
+        expanderClickHandler(displayedFacetField.field, e.altKey);
+    };
 
-    render() {
-        const {
-            facet,
-            displayedFacetField,
-            expanded,
-            expanderClickHandler,
-            selectedFacetTerms,
-            clearFacetSelections,
-            options,
-        } = this.props;
-        const maxTermCount = Math.max(...facet.terms.map((term) => term.count));
-        const labelId = `${displayedFacetField.field}-label`;
-        const FacetTermSelectRenderer = displayedFacetField.radio ? FacetTermRadio : FacetTermCheck;
-        return (
-            <div className="facet">
-                {!options.supressTitle
-                    ? (
-                        <FacetExpander
-                            title={facet.title}
-                            field={displayedFacetField.field}
-                            labelId={labelId}
-                            expanded={expanded}
-                            expanderEventHandler={displayedFacetField.nonCollapsable || !expanderClickHandler ? null : this.handleExpanderEvent}
-                            selectionCount={selectedFacetTerms.length}
-                            clearFacetSelections={displayedFacetField.radio ? null : clearFacetSelections}
+    const maxTermCount = Math.max(...facet.terms.map((term) => term.count));
+    const labelId = `${displayedFacetField.field}-label`;
+    return (
+        <div className="facet">
+            {!options.supressTitle
+                ? (
+                    <FacetExpander
+                        title={facet.title}
+                        field={displayedFacetField.field}
+                        labelId={labelId}
+                        expanded={expanded}
+                        expanderEventHandler={displayedFacetField.nonCollapsable || !expanderClickHandler ? null : handleExpanderEvent}
+                        selectionCount={selectedFacetTerms.length}
+                        clearFacetSelections={clearFacetSelections}
+                    />
+                )
+            : null}
+            {expanded || displayedFacetField.nonCollapsable ?
```

Comment:
In my local VS Code, I used tools other than ESLint, to spot code that can be written better.  One tool is sonarlint.  It barks at me when I use nested tertiary operators. The you have an opinion about nested tertiary operators?